src/helpers/WLClasses.hpp

```diff
@@ -20,12 +20,14 @@ AQUAMARINE_FORWARD(ISwitch);
 struct SRenderData {
     CMonitor* pMonitor;
     timespec* when;
-    double    x, y;
+    double    x = 0;
```

Comment:
these can be concat'd together: `x = 0, y = 0`

Reply:
changed